mlrun/api/db/sqldb/db.py

```diff
@@ -350,13 +353,65 @@ def list_artifact_tags(self, session, project):
         )
         return [row[0] for row in query]
 
-    def store_schedule(self, session, data):
-        sched = Schedule()
-        sched.struct = data
-        self._upsert(session, sched)
+    def create_schedule(
+        self,
```

Comment:
@Hedingber what about `state` , i.e. allowing to create a schedule in a draft/standby state

Reply:
Talked offline, for now adding the `state` and `desired_state` field to the models and schemas, the state handling mechanism will be added in a followup PR